### PR TITLE
chore(showcase): Adds www.projectival.de

### PR DESCRIPTION
**Pull request by sascha53**
Adds https://www.projectival.de/

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -4954,17 +4954,6 @@
   built_by: d:code:it
   built_by_url: https://dcodeit.com
   featured: false
-- title: SupremeKun - Street Fashion Marketplace and News
```
Comment (DSchau) [inline docs/sites.yml]: Why is this removed?

Comment (lannonbr) [reply docs/sites.yml]: It looks like this is not a Gatsby Site anymore. I'd say the removal would better to be in a separate PR rather than pushed into this PR

Comment (DSchau) [reply docs/sites.yml]: I should've validated 🤦‍♂️

I'll send a revert

Comment (DSchau) [reply docs/sites.yml]: Done in #12597
